code/app/build.gradle

```diff
@@ -57,8 +57,14 @@ dependencies {
     implementation project(':edgeidentity')
     implementation 'com.adobe.marketing.mobile:core:1+'
     implementation 'com.adobe.marketing.mobile:identity:1+'
-    implementation('com.adobe.marketing.mobile:edge:1+') {
+    implementation 'com.adobe.marketing.mobile:edgeconsent:1.+'
+    implementation('com.adobe.marketing.mobile:edge:1.3.2-SNAPSHOT') {
```

Comment:
We will keep Edge as pointing to Staging before merge to EdgeIdentity Staging?

Reply:
We should update this as soon as Edge 1.3.2 is released, we can make this change in dev branch before merging to staging.